app/views/users/index.html.erb

```diff
@@ -1,8 +1,7 @@
 <h1><%= t(".title") %></h1>
 <ul>
   <% @users.each do |user| %>
-    <li><%= user.name %></li>
-    <li><%= user.email %></li>
+    <%= render "detail", user: user %>
```

Comment:
If you have a `_user` partial in `app/views/users/`, you can just use:
```
<%= render @users %>
```
Because rails magically knows to look for a partial named `_user`

Reply:
Very cool! Done sha: 78a6e82